internal/core/server/commands/config.ts

```diff
@@ -63,6 +78,20 @@ async function runCommand(
 				),
 				...(Array.isArray(value) ? value : []),
 			]);
+		} else if (action === "remove") {
+			const lastKey = keyParts[keyParts.length - 1];
+			prevKeyConsumer.delete(lastKey);
+		} else if (action === "pop") {
+			const existing = Array.from(
+				keyConsumer.required([]).asIterable(),
+				(c) => c.asUnknown(),
+			);
+			const toRemove = Array.isArray(value) ? value : [];
```

Comment:
Shouldn't it be?

```suggestion
			const toRemove = Array.isArray(value) ? value : [value];
```

I don't understand the `[]` part.

Reply:
Should we rename `toRemove` with `valuesToRemove`?

Reply:
I forgot to add `[value].` Fixed.